cypress/integration/new.cy.js

```diff
@@ -47,17 +47,18 @@ describe('creating a new AO', () => {
     visTypes.forEach((visType) => {
```

Comment:
`navigates to the start page` can be consolidated as well? if so, remove the it and start each it with `goToStartPage`

Reply:
This refers to the code three lines above but the GH ui doesn't allow me to comment that line directly since its not within scope 🙈